packages/jellyfish-api-core/__tests__/category/masternode/getMasternodeBlock.test.ts

```diff
@@ -0,0 +1,94 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { RpcApiError } from '../../../src'
+
+describe('Masternode', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should getMasternodeBlocks', async () => {
+    const masternodeID = await client.masternode.createMasternode(await client.wallet.getNewAddress())
+
+    await container.generate(1)
+
+    const blocksResult = await client.masternode.getMasternodeBlocks({ id: masternodeID })
+
+    for (const value of Object.values(blocksResult)) {
+      expect(typeof value).toStrictEqual('string')
+      expect(value.length).toStrictEqual(64)
+    }
+  })
+
+  it('should getMasternodeBlocks with owner address', async () => {
```

Comment:
missing operator address??

Reply:
ownerAddress and operatorAddress are interchangeable that is why i felt it is redundant to add a separate test